installer/helm/chart/volcano/templates/webhooks.yaml

```diff
@@ -17,13 +17,18 @@ webhooks:
     failurePolicy: Fail
     matchPolicy: Equivalent
     name: mutatepod.volcano.sh
+{{- if .Values.custom.webhooks_namespace_selector }}
```

Comment:
If user set custom label, and pod in `Release.Namespace` will not be filtered, this is unacceptable.

Reply:
@Monokaix you think that it would be better if `Release.Namespace` would always be hardcoded? I guess it's a good idea to prevent accidental cluster break :D 

I'll try changing parameter to only specify additional namespaceSelector expressions with the expression 
```
        - key: kubernetes.io/metadata.name
          operator: NotIn
          values:
            - {{ .Release.Namespace }}
            - kube-system
```
always present then.

Reply:
Yes, you are right.

Reply:
Updated